dot/network/light.go

```diff
@@ -46,6 +109,12 @@ func NewLightRequest() *LightRequest {
 	}
 }
 
+func newLightRequestFromBytes(in []byte) (*LightRequest, error) {
+	msg := NewLightRequest()
+	err := msg.Decode(in)
```

Comment:
nit adding named returned values

```suggestion
func newLightRequestFromBytes(in []byte) (msg *LightRequest, err error) {
	msg = NewLightRequest()
	err = msg.Decode(in)
```